internal/page/attorney/sign_test.go

```diff
@@ -305,6 +337,7 @@ func TestPostSign(t *testing.T) {
 		appData         page.AppData
 		form            url.Values
 		lpa             *lpastore.Lpa
+		sendsToLpaStore bool
```

Comment:
This looks like its unused at the monent.

Reply:
:rage1: I thought I'd removed that